src/QMCWaveFunctions/BsplineFactory/SplineC2RAdoptor.h

```diff
@@ -184,7 +154,10 @@ struct SplineC2RSoA: public SplineAdoptorBase<ST,3>
   /** remap kPoints to pack the double copy */
   inline void resize_kpoints()
   {
+#ifndef QMC_CUDA
+    // GPU code needs the old ordering.
     nComplexBands=this->remap_kpoints();
```

Comment:
This comment is confusing define the ordering in the comment. Then define the new ordering.

Will the GPU code still work? It seems like if it does the i=i mapping is all the GPU code needs or is this just a case that it isn't caught by the unit tests?

Reply:
The comment is updated.
Yes. The GPU code works. The GPU kernel applying phase need the i-i mapping. Currently our uni tests are not covering the ordering and the integration tests are too small to have a difference in the order.